### ISSUE TITLE
Staging Review finding: Redundant statements of TTY phone number

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#86823]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VYE Team
**Product:** Verify Your Enrollment
**Feature:** Verify Your Enrollment

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the Need Help section at the bottom of [/mgib-enrollments](https://staging.va.gov/education/verify-school-enrollment/mgib-enrollments/) and other pages in the product, the text reads: "Call us at 888-442-4551 (TTY: 711). We’re here Monday through Friday, 8:00 a.m to 7:00 p.m ET. If you have hearing loss, call TTY: 711." There's no reason to state the TTY number twice.

## Recommended action

No preference on which TTY instance stays and which is removed. Users with hearing loss are likely to recognize what the TTY service is, so it's not necessary to explicitly state who TTY is for, but it's also okay to do so. Whichever TTY instance stays should still use the `va-telephone` component to be rendered as a link.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by alexisbetz** [comment]
https://jira.devops.va.gov/browse/VEBT-51
